hera_cal/frf.py

```diff
@@ -797,23 +1229,28 @@ def load_tophat_frfilter_and_write(datafile_list, baseline_list=None, calfile_li
         flag_yaml: path to manual flagging text file.
         skip_autos: bool, if true, exclude autocorrelations from filtering. Default is False.
                  autos will still be saved in the resides as zeros, as the models as the data (with original flags).
+        uvbeam: str, optional.
+            path to UVBeam object for calculating frate bounds.
+            default is None -> use other filter kwargs to determine fringe rate bounds.
+        verbose: bool, optional
+            Helpful text outputs.
         clean_flags_in_resid_flags: bool, optional. If true, include clean flags in residual flags that get written.
                                     default is True.
         filter_kwargs: additional keyword arguments to be passed to FRFilter.tophat_frfilter()
     '''
     if baseline_list is not None and Nbls_per_load is not None:
         raise NotImplementedError("baseline loading and partial i/o not yet implemented.")
-    hd = io.HERAData(datafile_list, filetype='uvh5', axis='blt')
+    hd = io.HERAData(datafile_list, filetype='uvh5')
     if baseline_list is not None and len(baseline_list) == 0:
         warnings.warn("Length of baseline list is zero."
                       "This can happen under normal circumstances when there are more files in datafile_list then baselines."
                       "in your dataset. Exiting without writing any output.", RuntimeWarning)
     else:
         if baseline_list is None:
             if len(hd.filepaths) > 1:
-                baseline_list = list(hd.bls.values())[0]
+                baseline_list = list(hd.antpairs.values())[0]
```

Comment:
This conflicts with the docstring, which describes baseline_list has being a list of antpairpols